x/evm/keeper/replay.go

```diff
@@ -73,17 +73,18 @@ func (k *Keeper) VerifyAccount(ctx sdk.Context, addr common.Address, accountData
 			panic(fmt.Sprintf("storage mismatch for address %s: expected %X, got %X", addr.Hex(), expectedState, actualState))
 		}
 	}
-	balance := accountData.Balance
+	// balance := accountData.Balance
 	nonce := accountData.Nonce
 	if !bytes.Equal(code, k.GetCode(ctx, addr)) {
 		panic(fmt.Sprintf("code mismatch for address %s", addr))
 	}
-	useiBalance := k.BankKeeper().GetBalance(ctx, k.GetSeiAddressOrDefault(ctx, addr), "usei").Amount
-	weiBalance := k.bankKeeper.GetWeiBalance(ctx, k.GetSeiAddressOrDefault(ctx, addr))
-	totalSeiBalance := useiBalance.Mul(sdk.NewInt(1_000_000_000_000)).Add(weiBalance).BigInt()
-	if balance.Cmp(totalSeiBalance) != 0 {
-		panic(fmt.Sprintf("balance mismatch for address %s: expected %s, got %s", addr.Hex(), balance, totalSeiBalance))
-	}
+	// we no longer check eth balance due to limiting EVM max refund to 150% of used gas (https://github.com/sei-protocol/go-ethereum/pull/32)
+	// useiBalance := k.BankKeeper().GetBalance(ctx, k.GetSeiAddressOrDefault(ctx, addr), "usei").Amount
+	// weiBalance := k.bankKeeper.GetWeiBalance(ctx, k.GetSeiAddressOrDefault(ctx, addr))
+	// totalSeiBalance := useiBalance.Mul(sdk.NewInt(1_000_000_000_000)).Add(weiBalance).BigInt()
+	// if balance.Cmp(totalSeiBalance) != 0 {
+	// 	panic(fmt.Sprintf("balance mismatch for address %s: expected %s, got %s", addr.Hex(), balance, totalSeiBalance))
+	// }
 	if nonce != k.GetNonce(ctx, addr) {
```

Comment:
do we want to leave these lines in comments for now?

Reply:
hm can delete them